ios-target/View/Home/HomeViewController.swift

```diff
@@ -7,30 +7,74 @@
 //
 
 import UIKit
+import MapKit
+import CoreLocation
 
 class HomeViewController: UIViewController {
   
   // MARK: - Outlets
   
-  @IBOutlet weak var welcomeLabel: UILabel!
-  @IBOutlet weak var logOut: UIButton!
+  @IBOutlet weak var mapView: MKMapView!
+  
+  let locationManager = CLLocationManager()
+  let regionMeters: Double = 1000
```

Comment:
This two props belong to the viewModel, also it might be a good idea to move location logic to your own manager to help reuse and avoid boiler plate on view controllers and/or view models.